pkg/test/harness.go

```diff
@@ -605,7 +605,9 @@ func (h *Harness) Report() {
 	if len(h.TestSuite.ReportFormat) == 0 {
 		return
 	}
-	if err := h.report.Report(h.TestSuite.ArtifactsDir, h.reportName(), report.Type(h.TestSuite.ReportFormat)); err != nil {
+
+	reportType := report.Type(strings.ToLower(h.TestSuite.ReportFormat))
```

Comment:
Instead of doing the change here, I think what should be modified is the `reportType` function from `pkg/kuttlctl/cmd/test.go` to store the report format in lower letters. That will help to not reproduce this problem in other places where `pkg/kuttlctl/cmd/test.go` is used.

Reply:
+1 agree with @iblancasa 
as this has been awhile from PR to review (my fault)... if I don't see an update today... I will merge (for proper attribution) and modify consistent to iblancasa comments.